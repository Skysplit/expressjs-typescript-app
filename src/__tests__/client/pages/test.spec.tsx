import React from 'react';
import { shallow } from 'enzyme';
import TestPage from '@client/pages/test';

describe('<HomePage />', () => {
  const setup = (props = {}) => {
    const wrapper = shallow(<TestPage {...props} />);
    return { wrapper, props };
  };

  describe('#render', () => {
    test('should display component', () => {
      const { wrapper } = setup();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
