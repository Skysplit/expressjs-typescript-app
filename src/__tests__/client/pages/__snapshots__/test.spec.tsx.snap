// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> #render should display component 1`] = `
<LinkRoutes
  route="home"
>
  <a>
    Home
  </a>
</LinkRoutes>
`;
